middleware/service_now.go

```diff
@@ -36,6 +36,9 @@ const defaultServiceNowUsername = "admin"
 const defaultServiceNowPassword = "providenow"
 const defaultServiceNowReachabilityTimeout = time.Second * 5
 
+const defaultServiceNowSchemaListPath = "api/x_ncurr_providenow/setup_configuration/ListSchemas"
```

Comment:
I still think this is overreach. We may need to just return not implemented if the environment variable is unset.